lib/facility.rb

```diff
@@ -1,14 +1,72 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_accessor :name, :address, :phone, :services, :registered_vehicles, :collected_fees, :hours, :closed_holidays
```

Comment:
Wondering why you made all of these accessors. My general rule of thumb is to keep them all as readers, then IF you need any as accessors, pull those out one by one. We want to restrict access to outside classes being able to change these values as much as possible.